src/test/mocks.utils/dynamo.db/mock.calls.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+
+import { DataMapper, QueryIterator, StringToAnyObjectMap } from '@aws/dynamodb-data-mapper';
+import { ConditionExpressionPredicate } from '@aws/dynamodb-expressions';
+import createMockInstance from 'jest-create-mock-instance';
+
+import { Nullable } from '../../../utils/types.utils';
+
+type ddbKeysCondition = {
+  [propertyName: string]: ConditionExpressionPredicate | any;
+};
+
+export function mockQuery<T, M>(
```

Comment:
now wraps Mocks in corresponding models, makes it easier to mimic key filter behavior